pandas/core/indexing.py

```diff
@@ -935,7 +938,11 @@ def _getitem_lowerdim(self, tup: tuple):
             #  is equivalent.
             #  (see the other place where we call _handle_lowerdim_multi_index_axis0)
             with suppress(IndexingError):
-                return self._handle_lowerdim_multi_index_axis0(tup)
+                # error "_LocationIndexer" has no attribute
+                # "_handle_lowerdim_multi_index_axis0"
+                return self._handle_lowerdim_multi_index_axis0(  # type: ignore[attr-defined]
```

Comment:
any ideas on how to make flake8&mypy&black accept this line (and the same line later in this file)?